#오늘 한 일
* git push origin 아이디 -> 에러 
에러 내용:
 ! [rejected]        chemainus04 -> chemainus04 (fetch first)
error: 레퍼런스를 'https://github.com/chemainus04/java-racingcar.git'에 푸시하는데 실패했습니다
힌트: 리모트에 로컬에 없는 사항이 들어 있으므로 업데이트가
힌트: 거부되었습니다. 이 상황은 보통 또 다른 저장소에서 같은
힌트: 저장소로 푸시할 때 발생합니다.  푸시하기 전에
힌트: ('git pull ...' 등 명령으로) 리모트 변경 사항을 먼저
힌트: 포함해야 합니다.

*찾아보다가 잘 모르겠어서 git status 하고 git push 하니까 성공함...
왜??pull 해야되는 거 아니었나? status 결과는 '현재 브랜치 step1' 이라고 뜸

pull request 
깃 공부

시간 복잡도 설명 -> http://skmagic.tistory.com/164
